File: Creational/Builder/Java/ConcreteProduct2.java

public class ConcreteProduct2 extends AbstractProduct{

	@Override
	public void stepA1() {
	}

	@Override
	public void stepA2() {
		this.string_set.add("stepA2");
	}

	@Override
	public void stepB1() {
	}

	@Override
	public void stepB2() {
		this.string_set.add("stepB2");
	}

	@Override
	public void stepC1() {
	}

	@Override
	public void stepC2() {
		this.string_set.add("stepC2");
	}

}

File: Creational/Builder/Python/ConcreteProduct2.py
from AbstractProduct import *

class ConcreteProduct2(AbstractProduct):

	def step_a_1(self):
		pass
		
	def step_a_2(self):
		self.string_set.add("step_a_2")
	
	def step_b_1(self):
		pass
	
	def step_b_2(self):
		self.string_set.add("step_b_2")
	
	def step_c_1(self):
		pass
	
	def step_c_2(self):
		self.string_set.add("step_c_2")